manuscript/03.results.md

```diff
@@ -2,60 +2,75 @@
 
 ### Lineage information provides unique information about the source and dynamics of heterogeneity
 
-To illustrate the unique value of lineage measurements when analyzing heterogeneous drug responses, we plotted a series of lineages from our measurements of the breast cancer cell line AU565 treated with gemcitabine (Fig. @fig:lineage). If we consider these to be two different, hypothetical populations, there is a striking difference in which cells contribute to the population’s growth but the starting and ending cell numbers are nearly identical. As a result, endpoint, population-level drug response measurements cannot distinguish these population structures. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to distinguish these two populations but make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to accurately quantify the extent and dynamics of heterogeneity. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel.
```

Comment:
Did you intend to make all these earlier changes?

Reply:
Many of these changes seem to revert edits I made. Please go through and check you intended each of these.